bedrock/mozorg/templates/mozorg/contribute/tasks/joy-of-coding.html

```diff
@@ -24,8 +24,8 @@ <h3>{{ _('Watch the intro.') }}</h3>
         </video>
         <button type="button" class="button-blue watch" data-action="play" data-task="joyofcoding" data-step="one" data-complete="true">{{ _('Watch the video') }}</button>
         {% if l10n_has_tag('contribute-tasks-0516') %}
-          <p><a class="more" rel="external" href="https://air.mozilla.org/channels/livehacking/">
-            {{ _('View more <em>Joy of Coding</em> videos on Air Mozilla') }}
+          <p><a class="more" rel="external" href="https://www.youtube.com/playlist?list=PLmaFLMwlbk8wKMvfEEzp9Hfdlid8VYpL5">
+            {{ _('View more <em>Joy of Coding</em> videos.') }}
```

Comment:
This link has a `»` added to it by the CSS so the addition of the period looks a little funny.

Reply:
```suggestion
            {{ _('View more <em>Joy of Coding</em> videos') }}
```